Sources/ComposableArchitecture/Effects/Animation.swift

```diff
@@ -0,0 +1,64 @@
+import Combine
+import SwiftUI
+
+extension Publisher {
```

Comment:
Given long-term thoughts on the `Effect` type, should we define this directly on `Effect` instead, or should we pull it out as a Combine helper and define it in `CombineSchedulers`, instead (though it admittedly isn't a scheduler-specific operation)?